src/results/viz.jl

```diff
@@ -0,0 +1,79 @@
+using Bonito
+using WGLMakie, GeoMakie, GraphMakie
+using Oxygen: html
+
+function setup_result_viz_routes()
```

Comment:
Same comments as before - not sure if these routes need to be defined at runtime.